src/components/layout/yearInReview/highlightBlock.tsx

```diff
@@ -17,7 +17,7 @@ export const HighlightBlock: React.FC<HighlightBlockProps> = ({
   children,
 }) => {
   return (
-    <div className='flex flex-col md:flex-row w-4/5 xl:w-2/3 2xl:w-2/3 mx-auto mb-8'>
+    <div className='flex flex-col md:flex-row xl:w-2/3 2xl:w-2/3 mx-auto mb-8'>
```

Comment:
does this effect usages of this component in other yir pages?

Reply:
Can this be removed or are you planning to use this prop? Maybe I'm viewing old code and you already removed it